identity/data_user.go

```diff
@@ -0,0 +1,56 @@
+package identity
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/databrickslabs/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+
+// DataSourceUser returns information about user specified by user name
+func DataSourceUser() *schema.Resource {
+	type entity struct {
+		UserName             	string   `json:"user_name"`
+		Home             	string   `json:"home" tf:"computed"`
+		Alphanumeric             	string   `json:"alphanumeric" tf:"computed"`
+	}
+
+	s := common.StructToSchema(entity{}, func(
+		s map[string]*schema.Schema) map[string]*schema.Schema {
+		// nolint once SDKv2 has Diagnostics-returning validators, change
+		s["user_name"].ValidateFunc = validation.StringIsNotEmpty
```

Comment:
We've migrated to other linter, so comment above might not work. You need ValidateDiagFunc and wrap this sdkv1 one

Reply:
I guess it's done now that I've changed to schema map.